tests/sources/math/test_vector4_32_mix_c.cpp

```diff
@@ -0,0 +1,31 @@
+////////////////////////////////////////////////////////////////////////////////
+// The MIT License (MIT)
+//
+// Copyright (c) 2018 Nicholas Frechette & Animation Compression Library contributors
+// Copyright (c) 2018 Nicholas Frechette & Realtime Math contributors
```

Comment:
Minor correction. This copyright line isn't accurate as the file was originally copy/pasted from here (or could have been). Double check the other added files as well but the date isn't an issue for me since they are copy/pasted from a file created in 2018.